Subject: avoid test failure in setup.test by hardcoding QUILT_PATCHES

Without this patch, quilt setup would try to use debian/patches instead
of patches and would fail. This is because quilt now initializes
QUILT_PATCHES by looking up parent directories and it would find
.pc/.quilt_patches of the debian source package.

The right upstream fix is probably to hardcode the location in quilt setup
but until that is approved, this fix is less intrusive and doesn't change
the behaviour of quilt setup.

Bug-Debian: http://bugs.debian.org/573689

--- a/test/setup.test
+++ b/test/setup.test
@@ -1,3 +1,4 @@
+$ export QUILT_PATCHES=patches
 $ mkdir dir
 
 $ echo 1 > dir/foo.orig
